lib/solid_queue/cli.rb

```diff
@@ -5,12 +5,12 @@
 module SolidQueue
   class Cli < Thor
     class_option :config_file, type: :string, aliases: "-c",
-      default: Configuration::DEFAULT_CONFIG_FILE_PATH,
+      default: ENV["SOLID_QUEUE_CONFIG"] || Configuration::DEFAULT_CONFIG_FILE_PATH,
```

Comment:
I was thinking that perhaps the easiest way would be not to provide any defaults here so that `Configuration` is the only place where we need to care about defaults 🤔

Reply:
That works, but the downside is that it no longer appears in `bin/jobs --help`.

Before:
```
bin/jobs -h
Commands:
  jobs help [COMMAND]  # Describe available commands or one specific command
  jobs start           # Starts Solid Queue supervisor to dispatch and perform enqueued jobs. Default command.

Options:
  -c, [--config-file=SOLID_QUEUE_CONFIG]                          # Path to config file
                                                                  # Default: config/queue.local.yml
      [--recurring-schedule-file=SOLID_QUEUE_RECURRING_SCHEDULE]  # Path to recurring schedule definition
                                                                  # Default: config/recurring.local.yml
      [--skip-recurring]                                          
                                                                  # Default: false
```

After
```
bin/jobs -h                                                                            
Commands:
  jobs help [COMMAND]  # Describe available commands or one specific command
  jobs start           # Starts Solid Queue supervisor to dispatch and perform enqueued jobs. Default command.

Options:
  -c, [--config-file=SOLID_QUEUE_CONFIG]                          # Path to config file
      [--recurring-schedule-file=SOLID_QUEUE_RECURRING_SCHEDULE]  # Path to recurring schedule definition
      [--skip-recurring]                                          
                                                                  # Default: false
```

Let me know if I should proceed with the changes.

Reply:
Ah, good point! Maybe we could just add this to the description? It's also that it's a bit misleading that the value of the env var is a default... 🤔  because if you unset that, the default will change 🤔 

I'm thinking of something like this: 
```diff
diff --git a/lib/solid_queue/cli.rb b/lib/solid_queue/cli.rb
index d36ae5f..947a86c 100644
--- a/lib/solid_queue/cli.rb
+++ b/lib/solid_queue/cli.rb
@@ -10,8 +10,7 @@ module SolidQueue
       banner: "SOLID_QUEUE_CONFIG"

     class_option :recurring_schedule_file, type: :string,
-      default: Configuration::DEFAULT_RECURRING_SCHEDULE_FILE_PATH,
-      desc: "Path to recurring schedule definition",
+      desc: "Path to recurring schedule definition (default: #{Configuration::DEFAULT_RECURRING_SCHEDULE_FILE_PATH}).",
       banner: "SOLID_QUEUE_RECURRING_SCHEDULE"
```

Reply:
Done:

```
bin/jobs -h
Commands:
  jobs help [COMMAND]  # Describe available commands or one specific command
  jobs start           # Starts Solid Queue supervisor to dispatch and perform enqueued jobs. Default command.

Options:
  -c, [--config-file=SOLID_QUEUE_CONFIG]                          # Path to config file (default: config/queue.yml).
      [--recurring-schedule-file=SOLID_QUEUE_RECURRING_SCHEDULE]  # Path to recurring schedule definition (default: config/recurring.yml).
      [--skip-recurring]                                          
                                                                  # Default: false
```